sdk/communication/Azure.Communication.CallAutomation/src/Models/Events/ReasonCode.cs

```diff
@@ -20,8 +20,12 @@ public ReasonCode(string value)
             _value = value ?? throw new ArgumentNullException(nameof(value));
         }
 
+        private const string RecognizeOperationcanceledValue = "8508";
```

Comment:
RecognizeOperationcanceledValue - is not needed we bucket this under  the canceled event

Reply:
yes, would remove it